GameDesign/Assets/Scripts/Events/GameEventTrigger.cs

```diff
@@ -31,6 +33,12 @@ public void TriggerEventViaCode(Collider other)
 
         if (EventTriggerCondition(other))
         {
+            Debug.LogError(PlayerPrefs.GetInt(m_event.name, 0));
+            if (!m_hasGameEventBeenTriggered && PlayerPrefs.GetInt(m_event.name, 0) != 1)
+            {
+                bossMutter = GetComponent<AudioSource>();
+                bossMutter.Play();
```

Comment:
You need to add a null check. not all events will have an audio source

Reply:
also technically you should have the playing of the audio on the listener, not on the trigger

Reply:
Who is the listener?